include/envoy/filter/http/filter_config_provider.h

```diff
@@ -32,11 +32,14 @@ class FilterConfigProviderManager {
    * @param filter_config_name the filter config resource name.
    * @param factory_context is the context to use for the filter config provider.
    * @param stat_prefix supplies the stat_prefix to use for the provider stats.
+   * @param last_filter_in_current_config indicate the filter position at filter chain
```

Comment:
maybe something like `indicates whether this filter is the last filter in the configured chain`

Reply:
fixed.